elifepubmed/utils.py

```diff
@@ -130,3 +130,14 @@ def abstract_parts(abstract, label_types):
         if part:
             parts.append(part)
     return parts
+
+
+def first_property(the_object, properties):
```

Comment:
When I first read this,  I found the name a little misleading. I would recommend renaming this function to `get_property` and updating the description to `Given a list of attributes returns the first non None value from an object attribute.`.

Reply:
I would tend to name some functions `get_` on occasion, but it's be suggested to me to not add `get_` to function names, just a little detail. I don't think we have a very strict policy on naming unless if we do please point me to it and I can follow what is recommended. Naming is hard!

It is causing me to review my naming choice, however, and `property` here I should say instead as `attribute` I think, since `getattr` clearly gets attributes and not properties.

Reply:
I would recommend giving this a quick glance, especially where it refers to class methods: https://medium.com/mindorks/how-to-write-clean-code-lessons-learnt-from-the-clean-code-robert-c-martin-9ffc7aef870c

Lectures by Robert C. Martin (Uncle Bob), one of the founders of Agile and SOLID methodologies, on YouTube are also good for learning about coding best practices from veteran developers.